docs/API.md

```diff
@@ -438,11 +438,14 @@ being acknowledged)
 | ---------------- | ----- | ------- | --------- | ------- | -------- | ------ |
 | `SRTO_INPUTBW`   | 1.0.5 | post    | `int64_t` | bytes/s | 0        | 0..    |
 
-- Sender nominal input rate. Used along with `OHEADBW`, when `MAXBW` is set to 
-relative (0), to calculate maximum sending rate when recovery packets are sent 
-along with main media stream (`INPUTBW * (100 + OHEADBW) / 100`). If `INPUTBW` 
-is not set while MAXBW is set to relative (0), the actual input rate is evaluated 
-inside the library.
+- This option is effective only if `SRTO_MAXBW` is set to 0 (relative). It
+controls the maximum bandwidth together with `SRTO_OHEADBW` option according
+to the formula: `MAXBW = INPUTBW * (100 + OHEADBW) / 100`. When this option
+is set to 0 (automatic) then the real INPUTBW value will be estimated from
+the sending rate during tranmission.
+
+- *Recommended: set this option to the predicted bitrate of your live stream
+and `SRTO_OHEADBW` leave with default 25%*
```

Comment:
> and `SRTO_OHEADBW` leave with default 25%*

"leave" should be before `SRTO_OHEADBW`.
Suggestion:
_"and use the default value of 25% for `SRTO_OHEADBW`"_

Reply:
I don't like it either - your version doesn't suggest that the value should remain unchanged, as well as "default" sounds ambiguous here.